src/components/content/order/create/CreateService.tsx

```diff
@@ -14,8 +14,9 @@ import { SelectServiceForm } from './SelectServiceForm';
 
 function CreateService(): React.JSX.Element {
     const [urlParams] = useSearchParams();
+    const [categoryName] = useState(location.hash.split('#')[1]);
```

Comment:
why do we need this in state? in this case, the browser URL is the state. we directly take it from there.

Reply:
fixed.